### DISCUSSION TITLE
Container image?

### DISCUSSION THREAD

**Discussion by static-moonlight** [versity/versitygw#390] [Q&A] [answered]
Hi, I'm planning to run this in my Kubernetes cluster. For that I would require a container image to deploy it. Temporary I could create such image myself, for evaluation and testing. But when I get closer to production, I would prefer an official image. Are there any plans to provide containerized builds in the future?

**Comment by benmcclelland** [comment]
We were planning on having docker containers for releases. Would this meet your needs if we had this hosted on dockerhub?

**Reply by benmcclelland** [reply, marked as answer]
A new release has been tagged with  the updates to generate container images for all new releases now. See https://github.com/versity/versitygw/wiki/Docker for the container details.

**Comment by static-moonlight** [comment]
> We were planning on having docker containers for releases. Would this meet your needs if we had this hosted on dockerhub?

Yes, absolutely. I'm not picky in terms of the registry. `docker.io`, `quay.io`, `ghcr.io` ... all of them are fine. However, just to let you know. Currently `ghcr.io/versity/versitygw:latest` requires authentication?! So for the moment I go with `docker.io/versity/versitygw:v0.13` (thanks for uploading it there as well). Whatever the final choice for your primary registry is gonna be, I will use that in the end. For now I am satisfied :)

Thank you so much!

**Reply by benmcclelland** [reply]
No problem. The automation is in place for both docker.io and ghcr.io for every new release.  I'll check into the ghcr.io authentication, and see if there is something I need to do there.
